packages/zen-bull-vault/src/FlashZen.sol

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: GPL-3.0-only
+//SPDX-License-Identifier: BUSL-1.1
```

Comment:
``StrategyMath.sol`` is imported here which is ``//SPDX-License-Identifier: AGPL-3.0-only ``, doesn't this mean we have to make it GPL?

It also imports our contract ``UniFlash.sol``, which I think needs to be GPL.  It also imports our contract ``VaultLib.sol, which is licensed as ``//SPDX-License-Identifier: GPL-2.0-or-later
``